ccmlib/cluster.py

```diff
@@ -251,18 +262,43 @@ def populate(self, nodes, debug=False, tokens=None, use_vnodes=False, ipprefix=N
             self.ipformat = ipformat
         elif not self.ipformat:
             self.ipformat = self.ipprefix + "%d"
-        node_count = nodes
         dcs = []
         self.use_vnodes = use_vnodes
-        if isinstance(nodes, list):
+        if isinstance(nodes, int):
+            node_count = nodes
+        elif isinstance(nodes, list):
             self.set_configuration_options(values={'endpoint_snitch': self.snitch})
             node_count = 0
             i = 0
             for c in nodes:
                 i = i + 1
                 node_count = node_count + c
                 for x in range(0, c):
-                    dcs.append('dc%d' % i)
+                    dcs.append(('dc%d' % i, None))
```

Comment:
dcs append here 3 element tuple, while below 2 element

Reply:
how 3?
`'dc%d' % i` and `None`

Reply:
I confused, sorry